File: W2/arrayScan.c
#include <stdio.h>

int arrayScan(FILE *in, int array[], int limit) {
    int quantity = 0;
    
    for ( ; quantity < limit && fscanf(in, "%d", &array[quantity]) == 1; quantity++ );
    return quantity;
}

int main() {
    int limit = 100;
    int array[limit];
    FILE *in = fopen("arrayScanIn.txt", "r");
    
    printf("%d\n", arrayScan(in, array, limit));
    
    fclose(in);
    
    return 0;
}

File: PHP/arrayScan.php
<?php
    function arrayScan($in, $length) {
        $outArray = fread($in, $length);
        $arrayElem = explode(" ", $outArray);
        
        return count($arrayElem);
    }
    
    $length = 1024;
    $in = fopen("arrayScan.txt", "r");
    
    echo arrayScan($in, $length) . PHP_EOL;
    
    fclose($in);
?>
